3236 - Fix multiple group photobook issue

crossan007 opened the pull request:
closes #3236 
@phili67  please let me know if this fixes the loop issue as you expect.

- loops over an array of input groups
- added a bottom margin, so that new pages are added if there are still more kiddos
- converted to using the full photos instead of thumbnails

phili67 commented:
Sorry but this doesn't fix completely my first code.
We've got some groups with 35 to 60 children.
The classes are split to 25 now.
Last the images were in high resolution, for a photobook it's better

crossan007 commented:
take a look now - should be fixed.

added a bottom margin, so that new pages are added if there are still more kiddos
converted to using the full photos instead of thumbnails

phili67 commented:
![capture d ecran 2017-11-10 a 19 23 57](https://user-images.githubusercontent.com/20263693/32672891-226235e4-c64d-11e7-8d01-7c49b46381b7.png)
You loose a line of photo hear.
and the frame isn't aline with the photo.
After it's perfect.

phili67 commented:
With :
//$this->initializeArrays();
        $this->pageMarginL = 15;
        $this->pageMarginR = 15;
        $this->pageMarginT = 20;
        $this->pageMarginB = 5;
        $this->personMarginL = 2.5;
        $this->personMarginR = 2.5;
        $this->personImageHeight = 30;
        $this->personImageWidth = 30;
        $this->FYIDString = MakeFYString($iFYID);

at the beginning and :

        if (file_exists($thumbnailURI)) {
            $this->Image($thumbnailURI, $this->currentX+.25, $this->currentY+.25, $this->personImageWidth-.5, $this->personImageHeight-.5, 'PNG');

It's perfect ....

Thank's a lot Charles and God bless you.

phili67 commented:
close #3236
